flink-sql-connector-hbase-2.2/pom.xml

```diff
@@ -39,15 +39,19 @@ under the License.
 			<groupId>org.apache.flink</groupId>
 			<artifactId>flink-connector-hbase-2.2</artifactId>
 			<version>${project.version}</version>
-			<exclusions>
-				<exclusion>
-					<groupId>org.apache.hbase</groupId>
-					<artifactId>hbase-client</artifactId>
-				</exclusion>
-			</exclusions>
 		</dependency>
 	</dependencies>
 
+	<dependencyManagement>
+		<dependencies>
+			<dependency>
+				<groupId>org.apache.hbase</groupId>
+				<artifactId>hbase-client</artifactId>
+				<version>${hbase2.version}</version>
+			</dependency>
```

Comment:
This entry shouldn't be necessary since `flink-connector-hbase-2.2` already sets the version to `hbase2.version`.
If it didn't we'd have bigger issues, because that'd mean users would have to do the same thing.